bin/compile

```diff
@@ -34,8 +34,8 @@ ENV_DIR=$3
 export BUILD_DIR CACHE_DIR ENV_DIR
 
 # Python defaults
-DEFAULT_PYTHON_VERSION="python-3.6.2"
-LATEST_3="python-3.6.2"
+DEFAULT_PYTHON_VERSION="python-3.6.3"
```

Comment:
Perhaps `DEFAULT_PYTHON_VERSION` should be moved below the `LATEST_*` variable assignments, and changed to `DEFAULT_PYTHON_VERSION="$LATEST_3"` to reduce duplication?